apps/api/src/.ormconfig.ts

```diff
@@ -4,7 +4,6 @@ import { DatabaseNamingStrategy } from './database/database.naming-strategy';
 dotenv.config();
 // Check typeORM documentation for more information.
 export const config: ConnectionOptions = {
-  name: 'default',
```

Comment:
I don't think this is the issue. perhaps we can set `connectTimeoutMS` to `10000`